earth_enterprise/BUILD_RHEL_CentOS.md

```diff
@@ -70,7 +68,6 @@ sudo yum install git-lfs
 To obtain GCC 4.8 on RHEL, you must install the "Development Tools" from RHSM: 
 ```
 sudo yum --setopt=group_package_types=mandatory,default,optional groupinstall "Development Tools"
```

Comment:
You can replace installing the full "Development Tools" group on RHEL and Cent OS 6 & 7 by:

```
yum install ant bzip2 doxygen gcc-c++ patch python-argparse python-setuptools swig tar

```

Reply:
done